### DISCUSSION TITLE
Proble with service centreontrapd

### DISCUSSION THREAD

**Discussion by jlfreon** [centreon-archive/centreon-archived#11319] [Q&A] [answered]
hello,
I warn I am a beginner; please detail the actions you will give me.
I have unpolished the image CENTOS07 VMWARE OVA Version 22.04
All was working fine but now I have this message:

The message is:
CRITICAL: Number of current processes running: 0

Can you help me to solve this problem
Thank you in advance

**Comment by ponchoh** [comment, marked as answer]
Moving the party over to 

[![thewatch](https://user-images.githubusercontent.com/85764737/177160295-7a347171-eb92-4c9e-af5a-6e4ebcea32ed.png)](https://thewatch.centreon.com/data-usage-and-visualization-38/problem-with-centreontrapd-788)

https://thewatch.centreon.com/data-usage-and-visualization-38/problem-with-centreontrapd-788
